Remotion/Web/Core/Res/Themes/NovaViso/HTML/WebButton.css

```diff
@@ -6,13 +6,13 @@
   --webbutton-min-height: var(--webbutton-min-height-standard);
 }
 
-button.webButton,
-input.webButton
+button.webButton
 {
   min-height: var(--webbutton-min-height);
   padding: 0;
   box-sizing: border-box;
   display: inline-block;
+  border-color: var(--color-primary);
```

Comment:
I think we need to do this in Common.css. The border look&feel is unified for all button elements via Common.css, so the special color applied to buttons needs to be implemented there as well.